lib/trip_dispatcher.rb

```diff
@@ -88,6 +90,57 @@ def load_trips
       end
 
       trips
+    end # end load_trips
+
+    def find_available_driver
+      @drivers.each do |driver|
+        if driver.status == :AVAILABLE
```

Comment:
This and `find_next_trip_id` are both great examples of helper methods. Good work demonstrating functional decomposition!